Content.Shared/GameObjects/Messages/Mob/HealthHud.cs

```diff
@@ -25,4 +27,20 @@ public enum ScreenEffects
         CircleMask,
         GradientCircleMask,
     }
+
+    [Serializable, NetSerializable]
+    public class LimbRender
+    {
+        public string Name;
+        public Color Color;
+
+        public LimbRender(string name, Nullable<Color> color = null)
+        {
+            Name = name;
+            if (color.HasValue)
```

Comment:
This makes it impossible to distinguish between an uninitialized value and "not colored". Either default initialize `Color` to like `White` in the else case or make `Color` also nullable.

Reply:
it fails to work since the code can't believe color can have null. it also says me that it shouldn't be Nullable since it can has null(?????????)

Reply:
You're missing the fact that `Color` cannot be null because it's a value type. `Nullable<T>` (or shorthand `T?` makes it have a "null" possibility), but you obviously cannot assign `Color?` to `Color` without handling the potential of a null.

If you want to be able to encode that the color of the `LimbRender` can be not set you should make the field type in this class nullable too.
Or you can just have it default to some sane default like white. Either works.